app/controllers/ems_common.rb

```diff
@@ -678,6 +699,38 @@ def refreshemss
     end
   end
 
+  def call_ems_disable(emss)
+    process_emss(emss, "disable_ems") unless emss.empty?
+    return if @flash_array.present?
+
+    add_flash(n_("Disable initiated for %{count} %{model} from the %{product} Database",
+                 "Disable initiated for %{count} %{models} from the %{product} Database", emss.length) %
```

Comment:
I think disabling multiple managers is a valid use case. To avoid the behavior we can go with separate buttons for `Enable selected Providers` and `Disable selected providers` when dealing with multiple select.